service/src/test/java/bio/terra/drshub/util/SignedUrlTestUtils.java

```diff
@@ -51,6 +51,7 @@ public class SignedUrlTestUtils {
   public static void setupSignedUrlMocks(
       AuthService authService,
       GoogleStorageService googleStorageService,
+      String ip,
```

Comment:
Since the `ip` isn't being used here could we remove it?

Reply:
it is marked as unused here but if I had to add it to get unit tests to pass